integration-tests/chopsticks/src/network/utils.ts

```diff
@@ -0,0 +1,53 @@
+export function getSiblingDestination(paraId: number) {
+	return {
+		parents: 1,
+		interior: {
+			X1: { Parachain: paraId },
+		},
+	}
+}
+
+export function getParentAccountDestination() {
+	return {
+		parents: 1,
+		interior: 'Here',
+	}
+}
+
+export function getAccountDestinationV2(addr: string) {
+	return {
+		V2: {
+			parents: 0,
+			interior: {
+				X1: {
+					AccountId32: {
+						network: 'Any',
+						id: addr,
+					},
+				},
+			},
+		},
+	}
+}
+
+export function getAccountDestinationV3(addr: string) {
+	return {
+		V3: {
+			parents: 0,
+			interior: {
+				X1: {
+					AccountId32: {
+						id: addr,
```

Comment:
You forgot the `network` here?

Reply:
In XCM version 3, network is optional. Any reason to provide it here?

Reply:
Ok then it's fine, I was not aware of that.